### PR TITLE
LIVE-2498 Update LLC dependencies - @taquito

### PR DESCRIPTION
**Pull request by haammar-ledger**
**Duplicating here as the PR template seems to have a glitch...**
Update taquito dependencies in LLC.
These dependencies are used by the Tezos integration.

### 📝 Description

Update taquito dependencies in LLC.
These dependencies are used by the Tezos integration.

### ❓ Context

- **Impacted projects**: `ledger-live-common` 
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-2498 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

N/A

### 🚀 Expectations to reach

- [x] **Add account / receive / send on LLD (linux)**
- [x] **Add account / receive / send on LLM (android)**
- [ ] **Add account / receive / send on LLM (iOS)**

### CODE REVIEW

**Comment by haammar-ledger** [comment]
Integration tests run started: https://github.com/LedgerHQ/ledger-live/actions/runs/2690104116

**Comment by haammar-ledger** [comment]
> Integration tests run started: https://github.com/LedgerHQ/ledger-live/actions/runs/2690104116

Integration tests all green \o/

**Review by hedi-edelbloute — APPROVED** [review]

File: libs/ledger-live-common/dependencies.md
Comment (gre) [inline libs/ledger-live-common/dependencies.md:50]: mmh why is it blocked now tho? i think this PR unblock this specific topic 🤔

Comment (henrily-ledger) [reply libs/ledger-live-common/dependencies.md:50]: I need to take time to test it on iOS, I remember we got some issues with iOS, but it's seems they replace libsodium with stable-lib so it should be alright

Comment (valpinkman) [reply libs/ledger-live-common/dependencies.md:50]: i think we did some magic when porting to the mono-repo so we could patch the libsodium bullshit.

https://github.com/LedgerHQ/ledger-live/blob/develop/package.json#L103

Not sure if it's related but worth checking it out

**Review by valpinkman — APPROVED** [review]

File: libs/ledger-live-common/dependencies.md
Comment (gre) [reply libs/ledger-live-common/dependencies.md:50]: i'm not sure if my original message was clear, but we need to remove the mention `| **BLOCKED BY LLM (ticket missing)**                               |` specifically because I can see this PR now updates it so this PR literally remove the blocker 🤔  (I mean, after the verification there are no such iOS issue.)

**Review by gre — DISMISSED** [review]
see conversation above

**Comment by gre** [comment]
@haammar-ledger you can try to rebase again and it should pass 🤞

**Comment by gre** [comment]
who can check it works on iOS?

**Comment by haammar-ledger** [comment]
> who can check it works on iOS?

@gre done by Henri :heavy_check_mark:

**Pull request merged**
